storage-resize-images/CHANGELOG.md

```diff
@@ -9,3 +9,7 @@ feature - Add new param for deleting the original image.
 ## Version 0.1.1
 
 fixed - Fixed bug where certain edge cases led to already resized image being resized again (issue #7).
+
+## Version 0.1.0
+
+Initial release of the _Resize Images_ extension.
```

Comment:
Consistent syntax - all other files in this PR used asterisk for italics.

Reply:
ping on this change too

Reply:
I'm not sure why it's showing like that for you, on mine (both GitHub & editor it's correct): 
![image](https://user-images.githubusercontent.com/842078/71614747-8473c800-2ba5-11ea-9dc5-f340df0fcde6.png)